packages/utils/src/fallbackSubgraph.ts

```diff
@@ -0,0 +1,80 @@
+import { GraphQLClient } from "graphql-request";
+import { Variables, RequestDocument } from "graphql-request/dist/types";
+import { Headers } from "graphql-request/dist/types.dom";
+
+import { Logger } from "./logger";
+import { createLoggingContext } from "./request";
+
+export type SubgraphSyncRecord = {
```

Comment:
This seemed like a fine way to organize things, but if we need to separate the sync record from the client, I can change the type to be organized such that the sync record is nested in another type that holds the client.
i.e.
```ts
type Subgraph = {
  client: GraphQLClient;
  uri: string;
  syncRecord: SubgraphSyncRecord;
}
```
or something like that. Lmk @LayneHaber

Reply:
I think this is fine, also the real time is the `Sdk<T>` type below